exp/runtime/topologymutation/walker_test.go

```diff
@@ -130,7 +130,7 @@ func Test_WalkTemplates(t *testing.T) {
 				CommonResponse: runtimehooksv1.CommonResponse{
 					Status: runtimehooksv1.ResponseStatusFailure,
 					Message: "no kind \"Unknown\" is registered for version \"controlplane.cluster.x-k8s." +
-						"io/v1beta1\" in scheme \"pkg/runtime/scheme.go:100\"",
```

Comment:
This issue was reported in slack where the path reported was "go/pkg/runtime.scheme.go". This can be safely changed without losing coverage. I'm currently digging to the bottom of how the path is generated to see why it might have ended up different here. 

@mtougeron

Reply:
I'm on MacOS

```
$> pwd
/Users/tougeron/src/go/src/github.com/mtougeron/cluster-api
$> env | grep GO
GOPATH=/Users/tougeron/src/go
```

FWIW, I just tried without setting my GOPATH and it worked as expected.

Reply:
Interesting - I wonder if any other tests are dependent on using go modules :thinking: